File: week2/pattern8.c
#include<stdio.h>
int main()
{
    int i,j,limit=7;
    for(i=1;i<=limit;i++)
    {
       
        for(j=1;j<=limit;j++)
        {
            if(i==1 || i==limit || i==j ||j==1|| j==limit || i+j==limit+1){
            printf("*");
            }else{
                printf(" ");
            }
        }
        printf("\n");
    }

}

File: Js/pattern8.js
const scanner=require ("prompt-sync")();
var num=scanner("enter a limit");
let i,j;
x="";
for(i=1;i<=num;i++){
    for(j=1;j<=i;j++){
       x=x+j;
       
    }
  console.log(x);
    x=""
}